tests/Microsoft.Fx.Portability.MetadataReader.Tests/TestAssembly.cs

```diff
@@ -101,6 +103,8 @@ public ResourceStreamAssemblyFile(string fileName)
             public string Version { get; }
 
             public Stream OpenRead() => s_assembly.GetManifestResourceStream(Name);
+
+            public bool SkipBinaryIfPackageExists { get { return false; } }
```

Comment:
Please stay consistent with the rest of the file; `SkipBinaryIfPackagesExist => false;`